nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -299,6 +307,47 @@ def get_dali_bpe_dataset(
     return dataset
 
 
+class ASRPredictionWriter(BasePredictionWriter):
+    def __init__(self, dataset, output_file: str, use_cer=False):
```

Comment:
If we are writing WER, might as well write CER too. Too expensive to do it separately. Also, will this write global WER / CER? Cause sample level metric will behave like average of averages and give wrong scores.

Reply:
This code does not support global wer/cer calculation. These are per sample and will be written per sample in the output file. We can add that support in another PR.